.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,7 +9,7 @@
 
 - [ ] Ran `tox` checks to avoid unnecessary CI fails:
     ```console
-    uvx tox -e all-checks,pytest
+    uvx tox
```

Comment:
Hey @tcoratger this command ends with serving the docs server, can you help exclude mkdocs serve somehow?